lib/utils.js

```diff
@@ -73,6 +75,7 @@ module.exports.processMethod = async function (msg, cfg) {
   const body = config.body || {};
   const followRedirect = cfg.followRedirect !== 'doNotFollowRedirects';
   const { auth } = cfg;
+  const requestTimeout = cfg.requestTimeoutPeriod ? parseInt(cfg.requestTimeoutPeriod, 10) : REQUEST_TIMEOUT;
```

Comment:
Probably it makes sense to validate the field whether it contains a number? 
E.g. 'ddd' field gives NaN. How does the library handle this case?